Makefile

```diff
@@ -1,22 +1,26 @@
-.PHONY: test dev-install install upgrade install-pip-tools lint
+.PHONY: test dev-install install upgrade lint
 
 test:
 	py.test
 
-install-pip-tools:
-	pip install pip-tools
-
-dev-install: install-pip-tools
-	pip-sync dev-requirements.txt requirements.txt
+dev-install:
+	pip install -r requirements/base.txt
+	pip install -r requirements/development.txt
```

Comment:
requirements/development.in should include requirements/base.txt so the package versions stay consistent; then you'll only need to install the second file here, since the first will be a subset of it.